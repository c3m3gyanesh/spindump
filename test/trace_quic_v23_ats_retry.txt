A test of the ATS QUIC implementation, using stateless retry. The client is Lsquic, the server is ATS. The following Lsquic command was used:

    . /lsquic/http_client -H quic.ogre.com -s quic.ogre.com:4434 -p /en/latest/admin-guide/introduction.en.html
