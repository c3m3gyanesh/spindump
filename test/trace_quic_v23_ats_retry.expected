UDP 172.16.114.251 <-> 67.192.246.159 54753:4443 at 1571173123450702 new starting packets 0 0 bytes 0 0 note "No response"
QUIC 172.16.114.251 <-> 67.192.246.159 0810cb787e0e691a-968b0e1da6e7a2689cb7b101d6c34a5dc48c50 (64882:4434) at 1571173128132446 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 67.192.246.159 10cb787e0e691a1d-1218e11624479e5bf64d7762145fa68429bd (64882:4434) at 1571173128332841 measurement starting left 599 packets 2 1 bytes 2456 121 bandwidth 2456 121 note "V23,no spin"
QUIC 172.16.114.251 <-> 67.192.246.159 10cb787e0e691a1d-12bd4bb466dfea8f3f8c9acab1a357b4a5c5 (64882:4434) at 1571173128544231 measurement up right 211390 packets 3 2 bytes 3684 242 bandwidth 3684 242 note "V23,no spin"
QUIC 172.16.114.251 <-> 67.192.246.159 10cb787e0e691a1d-12bd4bb466dfea8f3f8c9acab1a357b4a5c5 (64882:4434) at 1571173128964071 spinflip up 0-1 initiator packets 11 14 bytes 4667 13109 bandwidth 4667 13109 note "V23,no R-spin"
